### ISSUE TITLE
Integrate map generation

### ISSUE THREAD

**Issue by brendan-ward** [consbio/salcc_blueprint2#55]
`mbgl-renderer` node module has been added as a dependency, and `api/map.py` is a client for it.

See installation notes here:
https://github.com/consbio/mbgl-renderer#installation

Bring your dependencies up to date: `pipenv install` and `yarn install`

Make sure to put the mbtiles for the project in `/tiles`.

Once `yarn install` is complete (it will take a while), start up the server:
`./node_modules/.bin/mbgl-server -t /tiles`

Here is a minimal functional example using the map client.
```
import json

from map import *

unit_id = 'I1570'

data = json.loads(open('public/data/{0}.json'.format(unit_id)).read())
bounds = data['bounds']

# convert priorities to legend, in descending priority (dropping Not a Priority class)
priorities = json.loads(open('src/config/priorities.json').read())

legend = [(int(k), v['color'], v['label'])
          for k, v in priorities.items() if k != '0']
legend = sorted(legend, key=lambda x: x[0], reverse=True)
# strip off the value used for sorting
legend = [entry[1:] for entry in legend]

img = get_map_image(unit_id, bounds, legend)
# for testing only
img.save('api/tests/test_map.png')
```

The image save should get wrapped into a BytesIO buffer to send to report functions, so that we don't write any map files to disk.
